content/200-orm/200-prisma-client/600-observability-and-logging/250-opentelemetry-tracing.mdx

```diff
@@ -71,26 +69,28 @@ If your application sends a large number of spans to a [collector](https://opent
 To use tracing, you must do the following:
 
 1. [Install the appropriate dependencies](#step-1-install-up-to-date-prisma-orm-dependencies).
-1. [Enable the `tracing` feature flag in your Prisma schema file](#step-2-enable-the-feature-flag-in-your-prisma-schema-file).
-1. [Install OpenTelemetry packages](#step-3-install-opentelemetry-packages).
+1. [Install OpenTelemetry packages](#step-2-install-opentelemetry-packages).
+1. [Register tracing in your application](#step-3-register-tracing-in-your-application).
 
 ## Get started with tracing in Prisma ORM
 
 This section explains how to install and register tracing in your application.
 
 ### Step 1. Install up-to-date Prisma ORM dependencies
 
-Use version `4.2.0` or later of the `prisma`, `@prisma/client`, and `@prisma/instrumentation` npm packages.
+Use version `6.1.0` or later of the `prisma`, `@prisma/client`, and `@prisma/instrumentation` npm packages. You will also need to install the `@opentelemetry/api` package as it's a peer dependency.
 
 ```terminal
 npm install prisma@latest --save-dev
 npm install @prisma/client@latest --save
 npm install @prisma/instrumentation@latest --save
+npm install @opentelemetry/api@^1.8.0 --save
```

Comment:
```suggestion
npm install @opentelemetry/api@latest --save
```

Reply:
1.8.0 is the minimum supported but the users shouldn't think about it, the OTel ecosystem is on 1.9.0 now mostly